docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -217,14 +241,16 @@ In both cases, `quarkus-oidc` requires a connection to the specified OpenID Conn
 
 * If your Bearer tokens must be verified, use `quarkus-oidc`, `quarkus-smallrye-jwt`, or `quarkus-elytron-security-oauth2`.
 
-* If your Bearer tokens are in a JWT format, you can use either of the three extensions. Both `quarkus-oidc` and `quarkus-smallrye-jwt` support refreshing the JsonWebKey (JWK) set when the OpenID Connect provider rotates the keys.
+* If your Bearer tokens are in a JWT format, you can use either of the three extensions.
+Both `quarkus-oidc` and `quarkus-smallrye-jwt` support refreshing the JsonWebKey (JWK) set when the OpenID Connect provider rotates the keys.
 Therefore, if remote token introspection must be avoided or is unsupported by the providers, use `quarkus-oidc` or `quarkus-smallrye-jwt` for verifying JWT tokens.
 
 * If you need to introspect the JWT tokens remotely, you can use either `quarkus-oidc` or `quarkus-elytron-security-oauth2` because they support the verification of the opaque or binary tokens by using remote introspection.
 `quarkus-smallrye-jwt` does not support the remote introspection of both opaque or JWT tokens but instead relies on the locally available keys that are usually retrieved from the OpenID Connect provider.
 
-* `quarkus-oidc` and `quarkus-smallrye-jwt` support the injecting of JWT and opaque tokens into the endpoint code.
-Injected JWT tokens provide more information about the user. All extensions can have the tokens injected as `Principal`.
+* `quarkus-oidc` and `quarkus-smallrye-jwt` support the JWT and opaque tokens injection into the endpoint code.
+Injected JWT tokens provide more information about the user.
+All extensions can have the tokens injected as `Principal`.
```

Comment:
Will the target audience know what this means?

Reply:
I guess they will :) Thank you for the rev!